libs/core/shellbar/shellbar-branding/shellbar-branding.component.html

```diff
@@ -0,0 +1,20 @@
+<div
+    class="fd-shellbar__branding"
+    role="link"
+    tabindex="0"
+    [attr.aria-label]="ariaLabel"
+    (click)="callback ? callback($event) : ''"
+>
+    <span class="fd-shellbar__logo">
+        <ng-content select="fd-shellbar-logo"></ng-content>
+    </span>
+
+    <div class="fd-shellbar__product">
+        <span class="fd-shellbar__title">
+            <ng-content select="fd-shellbar-title"></ng-content>
+        </span>
```

Comment:
Same as above. This approach will result in elements having the same classes:
```
<span class="fd-shellbar__title">
    <fd-shellbar-title>
        <h1 class="fd-shellbar__title"> Portal</h1>
     </fd-shellbar-title>
</span>
```

`fd-shellbar__title` class is repeated twice and you have a span wrapping an h1 element.